admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.js

```diff
@@ -198,31 +216,42 @@ export default class CombinationsManager {
    * @private
    */
   watchEvents() {
-    this.eventEmitter.on(CombinationEvents.refreshCombinationList, () => this.refreshCombinationList(false));
-    this.eventEmitter.on(CombinationEvents.refreshPage, () => this.refreshPage());
-    this.eventEmitter.on(CombinationEvents.updateAttributeGroups, (attributeGroups) => {
-      const currentFilters = this.combinationsService.getFilters();
-      currentFilters.attributes = {};
-      Object.keys(attributeGroups).forEach((attributeGroupId) => {
-        currentFilters.attributes[attributeGroupId] = [];
-        const attributes = attributeGroups[attributeGroupId];
-        attributes.forEach((attribute) => {
-          currentFilters.attributes[attributeGroupId].push(attribute.id);
+    this.eventEmitter.on(CombinationEvents.refreshCombinationList, () => this.refreshCombinationList(false),
+    );
```

Comment:
@NeOMakinG Is it normal the return line ?

Reply:
I fixed it by disabling eslint on this block, but this is something we need to dive in in another PR (Eslint shouldn't do this on callbacks tbh)